Source/Core/VideoBackends/D3D/PerfQuery.h

```diff
@@ -8,14 +8,15 @@
 #include <d3d11.h>
 
 #include "VideoCommon/PerfQueryBase.h"
+#include "VideoBackends/D3D/D3DBase.h"
 
 namespace DX11
 {
 class PerfQuery : public PerfQueryBase
 {
 public:
   PerfQuery();
-  ~PerfQuery();
+  // TODO: end all queries (with EndQuery) in destructor?
```

Comment:
Yes. This comment is based on a comment in the PerfQuery::~PerfQuery, which was removed.